xml/hardening.xml

```diff
@@ -6550,7 +6581,7 @@ oracle           hard    nofile          63536</screen>
       this example up to the hard limit (in this example 63536) by
       executing:
      </para>
-<screen>ulimit -n 63536</screen>
+<screen>&prompt.root;ulimit -n 63536</screen>
```

Comment:
Make this screen and the above screens consistent with each other. Basically, use `<command>` to wrap your commands.